src/renderer/agents/mulmo_script.ts

```
@@ -0,0 +1,121 @@
+import { type AgentFunctionInfo, type AgentFunction } from "graphai";
+import { mulmoScriptSchema } from "mulmocast/browser";
+
+const mulmoScriptAgent: AgentFunction = async ({ namedInputs }) => {
+  const { arg } = namedInputs;
+  const { mulmoScript } = arg;
```

ESLint

[error] Remove this useless assignment to variable "mulmoScript".

(sonarjs/no-dead-store)